nfsstat/tests/compose/compose.yaml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+
+services:
+  nfs-server:
+    image: gists/nfs-server
+    container_name: nfs-server
+    privileged: true
+    ports:
+      - 2049:2049
+    volumes:
+      - ./nfs-share:/nfs-share
+
+  nfs-client:
```

Comment:
I don't even think we need the client as the agent is supposed to be the client.

Reply:
The Agent container doesn't start with enough privileged for us to mount a volume `mount -v -t nfs -o port=2049 nfs-server:/ /test1`. We could add some privileged there, but I thought it would be better to have a separate client container with enough privileged.

This is why we have a `privileged: true` in the docker-compose

Reply:
Ok fair enough, seems like the whole setup is probably quite different to how users would use the nfsstat integration from a containerized agent but this works well enough for the purpose of our tests. We don't have the tooling to start the agent as a privileged container at the moment.